src/AppBundle/Resources/views/Admin/Client/Report/manageConfirm.html.twig

```diff
@@ -1,5 +1,7 @@
 {% extends 'AppBundle:Layouts:application.html.twig' %}
 
+{% import 'AppBundle:Macros:macros.html.twig' as macros %}
```

Comment:
I'm curious about these imports as I presume they are already imported from `application.html.twig`?

Reply:
They are, and in Twig 1.x the import would have been passed down. In Twig 2.x however, imports are done at file level so you need to specifically import the macros file wherever it's used.

This means that:
- Import sources are clearer
- Twig files can be reused more confidently

It's probably also worth noting that, whilst I had to do this in a bunch of places, _most_ files were already importing the macros file even though they didn't technically need to.

Reply:
Good to know, thanks